code/drasil-lang/lib/Language/Drasil/Sentence.hs

```diff
@@ -51,12 +52,14 @@ data RefInfo = None
 infixr 5 :+:
 data Sentence where
   -- | Ch looks up the term for a given 'UID' and displays the term with a given 'SentenceStyle' and 'CapitalizationRule'.
-  -- This allows Sentences to hold plural forms of 'NounPhrase's and 'NamedIdea's.
+  -- This allows Sentences to hold plural forms of 'NamedIdea's.
   Ch    :: SentenceStyle -> TermCapitalization -> UID -> Sentence
   -- | A branch of Ch dedicated to SymbolStyle only.
   SyCh  :: UID -> Sentence
   -- | Converts a unit symbol into a usable Sentence form.
   Sy    :: USymb -> Sentence
+  -- | Directly embeds a 'NP'
+  NP    :: NP -> Sentence
```

Comment:
Do we ever want to permit this without a chunk being created first? On second thought, that could be its purpose!

Reply:
Yes, this is the case where we have the noun phrase in hand but never made a full chunk of it. It is used rarely, but simplified some code. Over time, we'll probably want to deprecate this.

Reply:
That sounds good to me.